docs/app/pages/components/sections/panels/item-display-panel-inline/item-display-panel-inline.component.ts

```diff
@@ -0,0 +1,152 @@
+import { Component, ViewChild, ViewChildren, TemplateRef, QueryList, ContentChild } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+import { IPlunk } from '../../../../../interfaces/IPlunk';
+import { IPlunkProvider } from '../../../../../interfaces/IPlunkProvider';
+import { ItemDisplayPanelComponent } from '../../../../../../../src/index';
+
+@Component({
+    selector: 'uxd-item-display-panel-inline-component',
+    templateUrl: './item-display-panel-inline.component.html',
+    styleUrls: ['./item-display-panel-inline.component.less'],
+    host: {
+        '(window:keydown.arrowup)': 'upArrow($event)',
+        '(window:keydown.arrowdown)': 'downArrow($event)'
+    }
+})
+@DocumentationSectionComponent('ComponentsItemDisplayPanelInlineComponent')
+export class ComponentsItemDisplayPanelInlineComponent extends BaseDocumentationSection implements IPlunkProvider {
+
+    @ViewChild('modalDoc') modalDoc: TemplateRef<any>;
+    @ViewChild('modalPpt') modalPpt: TemplateRef<any>;
+    @ViewChild('modalPdf') modalPdf: TemplateRef<any>;
+  
+    visible: boolean = false;
+    selectedItem: DisplayPanelItem;
+    previousEnabled: boolean = true;
+    nextEnabled: boolean = true;
+    shadow: boolean = false;
+    animate: boolean = true;
```

Comment:
Is this required anymore? If it is never set to false then we don't need it (i haven't checked everywhere so not sure)